homeassistant/const.py

```diff
@@ -146,6 +146,7 @@
 CONF_TTL = 'ttl'
 CONF_TYPE = 'type'
 CONF_UNIT_OF_MEASUREMENT = 'unit_of_measurement'
+CONF_UNIT_OF_MEASUREMENT_TEMPLATE = 'unit_of_measurement_template'
```

Comment:
This should not be in const.py because it is only used in one place. - please move it to the template sensor.

Reply:
Ok thanks, I have removed it from the const.py and moved it over to template.py. 

I suppose it will likely move back to the const.py once the unit_of_measure_template is introduced in the other template components.